docs/client-libraries-java-use.md

```diff
@@ -0,0 +1,123 @@
+---
+id: client-libraries-java-use
+title: Use a Java client
+sidebar_label: "Use"
+---
+
+## Create a producer
+
+Once you've instantiated a {@inject: javadoc:PulsarClient:/client/org/apache/pulsar/client/api/PulsarClient} object, you can create a {@inject: javadoc:Producer:/client/org/apache/pulsar/client/api/Producer} for a specific Pulsar [topic](reference-terminology.md#topic).
+
+```java
+Producer<byte[]> producer = client.newProducer()
+        .topic("my-topic")
+        .create();
+
+// You can then send messages to the broker and topic you specified:
+producer.send("My message".getBytes());
+```
+
+By default, producers produce messages that consist of byte arrays. You can produce different types by specifying a message [schema](#schema).
+
+```java
+Producer<String> stringProducer = client.newProducer(Schema.STRING)
+        .topic("my-topic")
+        .create();
+stringProducer.send("My message");
+```
+
+> Make sure that you close your producers, consumers, and clients when you do not need them.
+
+> ```java
+> producer.close();
+> consumer.close();
+> client.close();
+> ```
+
+>
+> Close operations can also be asynchronous:
+
+> ```java
+> producer.closeAsync()
+>    .thenRun(() -> System.out.println("Producer closed"))
+>    .exceptionally((ex) -> {
+>        System.err.println("Failed to close producer: " + ex);
+>        return null;
+>    });
+> ```
+
+## Create a consumer
+
+In Pulsar, consumers subscribe to topics and handle messages that producers publish to those topics. You can instantiate a new [consumer](reference-terminology.md#consumer) by first instantiating a {@inject: javadoc:PulsarClient:/client/org/apache/pulsar/client/api/PulsarClient} object and passing it a URL for a Pulsar broker (as [above](#client-configuration)).
```

Comment:
Acknowledgment basic template should look different based on type of subscription and if topic is partitioned or not.

If subscription is exclusive, the developer should use cumulative acknowledgment.
If subscription is exclusive, and topic is partitioned topic, there should be special code pattern to do cumulative ACK per partition.
I think it was @BewareMyPower or @tisonkun talking about it in the mailing list. 

For sure it should be in this page.